src/catalog/category-hidden.md

```diff
@@ -4,24 +4,27 @@ title: Hidden Categories
 
 There are many ways to use hidden categories. You might want to create additional category levels for your own internal purposes, but show only the higher-level categories to your customers. Or, you might want to link to a category that is not included in the navigation menu.
 
-## To create hidden categories:
+## Create hidden categories
 
 1. On the _Admin_ sidebar, go to **Catalog** > **Categories**.
 
-1. In the category tree, select the category you want to hide. Then, do the following:
+1. In the category tree, select the category you want to hide and do the following:
 
-   * Set **Is Active** to `Yes`.
-   * Set **Include in Menu** to `No`.
+   - Set **Is Active** to `Yes`.
+   - Set **Include in Menu** to `No`.
 
 1. In the **Display Settings** section, set **Anchor** to `No`.
 
     The hidden category is active, but does not appear in the top menu, or in layered navigation.
 
-1. Although the category is hidden, you can still create subcategories beneath it, and make them active. Complete the following settings for each hidden subcategory:
+1. Complete the following settings for each hidden subcategory to create subcategories:
 
-   * Set **Enable Category** to `Yes`.
-   * In the **Display Settings** section, set **Anchor** to `Yes`.
+   {:.bs-callout-info}
```

Comment:
for the bullets to show in the info box, don't you have to use the <br />?

Reply:
The bullets aren't meant to go inside of the note.